Preston_Kellen/Assignments/Dojo_Survey_Validate/dojosurvey.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, render_template, request, redirect, session, flash
+app = Flask(__name__)
+app.secret_key = 'ThisIsSecret'
+
+@app.route('/')
+def index():
+    return render_template("home.html")
+
+@app.route('/users', methods=['POST'])
+def create_user():
+    if len(request.form['name']) < 1:
+        flash("Please enter your name!")
+        return redirect('/')
+    else:
```

Comment:
No need for an `else:` statement since your `if` conditional has a return. In other words, _IF_ your conditional is true, code execution will not continue; as a corollary, _IF_ the conditional is false, it'll automatically continue code execution